types/felt.go

```diff
@@ -7,106 +7,147 @@ import (
 	"fmt"
 	"io"
 	"math/big"
+	"regexp"
 	"strconv"
 	"strings"
 )
 
+const (
+	FIELD_GEN   int    = 3
+	FIELD_PRIME string = "3618502788666131213697322783095070105623107215331596699973092056135872020481"
+)
+
+var (
+	MaxFelt   = StrToFelt(FIELD_PRIME)
+	utfRegexp = regexp.MustCompile(`\w+`)
+)
+
+// Felt represents Field Element or Felt from cairo.
 type Felt struct {
 	*big.Int
 }
 
-func (f *Felt) Scan(src interface{}) error {
-	var i sql.NullString
-	if err := i.Scan(src); err != nil {
-		return err
-	}
-	if !i.Valid {
-		return nil
+// Big converts a Felt to its big.Int representation.
+func (f *Felt) Big() *big.Int {
+	return new(big.Int).SetBytes(f.Int.Bytes())
```

Comment:
```suggestion
	return f.Int
```

Reply:
This method is more for avoiding a shallow copy of the pointer and instead getting a new big Int to manipulate. Maybe needs a more descriptive method name